generic_config_updater/gu_common.py

```diff
@@ -155,6 +159,32 @@ def validate_field_operation(self, old_config, target_config):
                 if any(op['op'] == operation and field == op['path'] for op in patch):
                     raise IllegalPatchOperationError("Given patch operation is invalid. Operation: {} is illegal on field: {}".format(operation, field))
 
+        def _invoke_validating_function(cmd):
+            # cmd is in the format as <package/module name>.<method name>
+            method_name = cmd.split(".")[-1]
+            module_name = ".".join(cmd.split(".")[0:-1])
+            module = importlib.import_module(module_name, package=None)
+            method_to_call = getattr(module, method_name)
+            return method_to_call()
+
+        if os.path.exists(UPDATER_CONF_FILE):
+            with open(UPDATER_CONF_FILE, "r") as s:
+                updater_conf = json.load(s)
+        else:
+            raise GenericConfigUpdaterError("GCU Config file not found") 
+        
+        for element in patch:
+            path = element["path"]
+            table = re.search(r'\/([^\/]+)(\/|$)', path).group(1)
```

Comment:
What does it match? Could you add an commented example?

Reply:
This matches the table name "PFC_WD" and removes any forward slashes at the beginning or end. I added a comment to clarify as well